mangaki/mangaki/tests/test_merge.py

```diff
@@ -0,0 +1,62 @@
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.contrib.auth import get_user_model
+from django.contrib import admin
+from django.db import connection
+from mangaki.models import Work, Editor, Category, Studio, WorkCluster, Rating
+from datetime import date, timedelta
+
+
+class MergeTest(TestCase):
+
+    def setUp(self):
+        # FIXME: The defaults for editor and studio in Work requires those to
+        # exist, or else foreign key constraints fail.
+        Editor.objects.create(pk=1)
+        Studio.objects.create(pk=1)
+
+        self.user = get_user_model().objects.create_superuser(username='test', password='test', email='steins@gate.co.jp')
+        self.users = []
+        for username in 'ABCD':
+            self.users.append(get_user_model().objects.create_user(username=username, password='test'))
+
+        today = date.today()
+        yesterday = date.today() - timedelta(1)
+        tomorrow = date.today() + timedelta(1)
+
+        anime = Category.objects.get(slug='anime')
+        Work.objects.bulk_create([Work(title='Sangatsu no Lion', category=anime) for _ in range(10)])
```

Comment:
Unlimited Duplicates Works. Shouldn't we create *different* animes and assume that duplicates may throw an `IntegrityError` ?